Jenkinsfile

```diff
@@ -178,6 +182,7 @@ pipeline{
 					sh "rm -r databases logs data reports orthopairs"
 					sh "rm -rf graph-importer*"
 					sh "rm -rf graph-qa*"
+					sh "rm -rf release-jenkins-utils*"
```

Comment:
I just realized, if there is an error with any command before these, will the removal of the repos not be run and so the clone commands will fail on a second run?  Maybe logic is needed to clone if a repo doesn't exist and pull if it does?